tests/test_anim.py

```diff
@@ -70,6 +70,19 @@ def test_delta_parsing(input_genomes_tiny: Path) -> None:
         method_anim.parse_delta(Path("/dev/null"))
 
 
+def test_bad_alignments_parsing(input_genomes_bad_alignments: Path) -> None:
+    """Check parsing of test NUCmer .delta/.filter file."""
+    assert method_anim.parse_delta(
+        input_genomes_bad_alignments
+        / "intermediates/ANIm/MGV-GENOME-0264574_vs_MGV-GENOME-0357962.filter",
+    ) == (
+        None,
+        None,
+        None,
+        None,
+    )
+
+
```

Comment:
If you drop the last comma, you can probably have ``(None, None, None, None)`` on one line?